### PR TITLE
MOOC-3076 Feat: add checking if course has Power form

### PR DESCRIPTION
**Pull request by nazgut**
MOOC-3076 add checking if course has Power form

### CODE REVIEW

File: navoica_api/api/v1/views.py [hunk 1]
```diff
@@ -633,3 +633,39 @@ class CareerViewSet(viewsets.ReadOnlyModelViewSet):
...
+
+    authentication_classes = (
+        OAuth2AuthenticationAllowInactiveUser, SessionAuthenticationAllowInactiveUser, JwtAuthentication
+    )
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, course_id=None):
+        course_key = get_course_key(request, course_id)
+        course = modulestore().get_course(course_key)
+        if course.other_course_settings.get('external_enroll') or course.other_course_settings.get('external_enroll').get('value'):
```
Comment (hoffmannkrzysztof) [inline navoica_api/api/v1/views.py]: its better instead of using if: just make an assignment to variable to power = course....

**Comment by hoffmannkrzysztof** [comment]
1. Wrong destination branch to merge
2. Incorrect title of PR

**Comment by nazgut** [comment]
Ad 1 changed
Ad 2 changed

**Comment by hoffmannkrzysztof** [comment]
@nazgut Brak wg zadania: 7. Adres rejestracji zdalnej

File: navoica_api/api/v1/views.py [hunk 2]
```diff
@@ -633,3 +633,39 @@ class CareerViewSet(viewsets.ReadOnlyModelViewSet):
...
+    )
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request, course_id=None):
+        course_key = get_course_key(request, course_id)
+        course = modulestore().get_course(course_key)
+        if course.other_course_settings.get('external_enroll').get('value'):
+            power = course.other_course_settings.get('external_enroll_url').get('value')
+        power = False
+        return Response({"power": power})
```
Comment (hoffmannkrzysztof) [inline navoica_api/api/v1/views.py]: power is always False

**Review by hoffmannkrzysztof — CHANGES_REQUESTED** [review]

**Pull request merged**